weighted.py

```diff
@@ -17,10 +17,11 @@ def roll(dice_sides):
 
 
 def main():
-    print("Rolling dice...")
-    result = roll(dice.sides)
-    print("Result:", result)
-
+    try:
+        sides = json.load(open(sys.argv[1]))
+    except:
```

Comment:
Should catch a particular error, probably IOError. Better thing is to use argparse.